jupyter-book/trajectories/rna_velocity.ipynb

```diff
@@ -15,53 +15,117 @@
    ]
```

Comment:
<p>"Instead of studying results in a low dimensional representation, it is suggested to work in a higher dimensional space instead. CellRank <strong>[CITE]</strong> has recently been proposed to automatically infer initial and terminal states, as well as fate probabilities based on RNA velocity."</p><p> </p><p>Not sure whether I like this as a key takeaway. If I were to read this I would not immediately know WHY one should work in high dim space. The CellRank suggestion is also a bit random. I would suggest that you discuss/show this in the actual text above instead. Happy to reference CellRank (and other tools that do similar things), but maybe not here.</p><p> </p><p>"Recent attempts to improve RNA velocity inference beyond the <em>EM model</em> include an automised way of quantifying if individual genes but also entire datasets are suitable for RNA velocity analysis <strong>[CITE]</strong>. Additionally, a novel approach for projecting the velocity stream has been proposed <strong>[CITE]</strong>.</p><p>"</p><p> </p><p>Should also not be in the key takeaways.</p><p>Here's another suggestion: Add a "new directions" section like Zoe &amp; Matthew did -&gt; https://www.sc-best-practices.org/trajectories/lineage_tracing.html#new-directions</p><p> </p><p>There you could discuss cellrank, veloVI and other tools that might be very recent and worthwhile discussing.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/multimodal-single-cell-best-practices/pull/100/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>